### DISCUSSION TITLE
Errore Assenze

### DISCUSSION THREAD

**Discussion by CobraOfficial01** [iisgiua/giuaschool#681] [Richieste di aiuto]
salve come posso risolvere l'errore allegato.
![image](https://github.com/iisgiua/giuaschool/assets/128899928/3c15a103-4007-48fd-a18d-5e403e604296)

**Comment by CobraOfficial01** [comment]
@trinko

**Comment by CobraOfficial01** [comment]
e qusto è il log di tutti i problemi
[2024-03-28_11-02.log](https://github.com/iisgiua/giuaschool/files/14791127/2024-03-28_11-02.log)

**Comment by CobraOfficial01** [comment]
@trinko

**Comment by trinko** [comment]
Spostato [qui](https://github.com/iisgiua/giuaschool/issues/682)
